test/spec/modules/userId_spec.js

```diff
@@ -25,7 +25,6 @@ import {
 import {server} from 'test/mocks/xhr.js';
 import find from 'core-js-pure/features/array/find.js';
 import {unifiedIdSubmodule} from 'modules/unifiedIdSystem.js';
-import {pubCommonIdSubmodule} from 'modules/pubCommonIdSystem.js';
```

Comment:
@mmoschovas pubCommonIdSystem.js is removed from the tests.

Reply:
@SKOCHERI I'm referring to test/spec/modules/sharedIdSystem_spec.js  which is using the pubCommonId.js functions not sharedIdSystem.js

Reply:
@mmoschovas In this PR, the pubCommonIdSystem.js is deleted but pubCommonId.js  is as is.

Reply:
@mmoschovas The test/spec/modules/sharedIdSystem_spec.js is updated. Can you review it?